src/h5web/vis-packs/core/visualizations.ts

```diff
@@ -119,4 +125,22 @@ export const CORE_VIS: Record<Vis, CoreVisDef> = {
       );
     },
   },
+
+  [Vis.RgbImage]: {
+    name: Vis.RgbImage,
+    Icon: FiMap,
+    Toolbar: RgbImageVisToolbar,
+    Container: RgbImageVisContainer,
+    ConfigProvider: RgbImageVisConfigProvider,
+    supportsDataset: (dataset) => {
+      const classAttr = getAttributeValue(dataset, 'CLASS');
+      return (
+        typeof classAttr === 'string' &&
+        classAttr === 'IMAGE' &&
+        hasArrayShape(dataset) &&
+        dataset.shape.length === 3 &&
+        hasIntegerType(dataset)
```

Comment:
The `supportsDataset` function is quite restrictive for now. I hope to expand it to float types in a future PR